experiments/long_runs/bucket.jl

```diff
@@ -106,16 +106,23 @@ domain =
 
 # Parameters
 params = LP.LandParameters(FT)
-σS_c = FT(0.2)
-W_f = FT(0.2)
-z_0m = FT(1e-3)
-z_0b = FT(1e-3)
-κ_soil = FT(1.5)
-ρc_soil = FT(2e6)
+
+name_map = (;
```

Comment:
Could this map live in src/toml also? otherwise we might have to include the same map in every bucket run

Reply:
nice! That looks way cleaner. (I think it's unavoidable that some kind of name map is present, though, since the names in the ClimaParams dont match our names in the structs).

having the constructor work with the toml_dict and work with the name map there sounds like a good idea!